### PR TITLE
Modernize

### PR DESCRIPTION
**Pull request by loic**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,32 +42,34 @@
     "nomnom": "^1.8.1",
     "object-assign": "^4.1.0",
     "on-finished": "^2.3.0",
+    "pouchdb": "newlogic42/dist-pouchdb",
```
Comment (loic) [inline package.json]: `pouchdb` build that includes the https://github.com/pouchdb/pouchdb/pull/6793 fix.

**Comment by marten-de-vries** [comment]
Great that you managed to get the tests to pass with more packages in the monorepo, and with a modern version of PouchDB. I think that in the long term, we could probably get rid of all the ``file:...`` dependencies and take the same approach as for the pouchdb repo by moving ``/packages/node_modules`` to ``/node_modules``. That said, I'm all for merging this into pouchdb-server as soon as we can fix the 'linking to an unreleased pouchdb' problem.

**Comment by loic** [comment]
@marten-de-vries do you see any downside to the `file:...` approach? From where I stand it describes precisely what's required to build the monorepo.

Did you get any feedback re the beta release hosted on npm?

**Comment by marten-de-vries** [comment]
> @marten-de-vries do you see any downside to the file:... approach? From where I stand it describes precisely what's required to build the monorepo.

@loic It seems fine to me, just pointing out that long-term it might be less work to have exactly the same configuration for both repos. It would prevent having to duplicate maintenance work on e.g. the automated publish script.

---

> Did you get any feedback re the beta release hosted on npm?

No, I haven't got an IRC response yet. I thought I'd look through the release code and then just try it for myself, but noticed I'm not personally an npm author for all the packages (especially, pouchdb). Time to try it this way:

@daleharvey @willholley @garrensmith (as the most recent pouchdb releasers): is running 'BETA=1 npm run release' a valid strategy to give @loic a patched pouchdb dependency for pouchdb-server without inconveniencing others? If so, could one of you do so for the latest PouchDB master? Thanks!

**Comment by loic** [comment]
Dear all, gentle reminder.

**Comment by marten-de-vries** [comment]
Sorry, I haven't had the time to follow through on this. Anyway, with PouchDB's recent 6.4.0 this should be unblocked.

**Comment by loic** [comment]
Thanks @marten-de-vries, I plan to resume this effort in the next couple of days.

**Comment by loic** [comment]
@marten-de-vries @gr2m created a new PR against `pouchdb-server`: https://github.com/pouchdb/pouchdb-server/pull/274

That should be enough to get a running build against PouchDB 6.4.0. Once this is merged I'm planning to focus on merging the remaining dependencies to the monorepo.

**Comment by loic** [comment]
Reopening for CI.

**Comment by gr2m** [comment]
I’m all for moving forward with this :)

**Pull request merged**